prime-router/metadata/schemas/covid-19.schema

```diff
@@ -1021,32 +1021,32 @@ elements:
   - name: test_authorized_for_otc
     type: TABLE
     table: LIVD-SARS-CoV-2
-    tableColumn: is_otc
+    tableColumn: Over the Counter (OTC) Home Testing
     reference: Is the test authorized for over-the-counter purchase by the FDA
     referenceUrl: https://www.fda.gov/news-events/fda-newsroom/press-announcements
-    mapper: livdLookup()
+    mapper: LivdYesNo()
     default: N
-    documentation: Is the test authorized for over-the-counter purchase by the FDA (Y, N, UNK)
+    documentation: Is the test authorized for over-the-counter purchase by the FDA (Y, N)
 
   - name: test_authorized_for_home
     type: TABLE
     table: LIVD-SARS-CoV-2
-    tableColumn: is_home
+    tableColumn: Over the Counter (OTC) Home Testing
     reference: Is the test authorized for home use by the FDA
     referenceUrl: https://www.fda.gov/news-events/fda-newsroom/press-announcements
-    mapper: livdLookup()
+    mapper: LivdYesNo()
     default: N
     documentation: Is the test authorized for home use by the FDA (Y, N, UNK)
 
   - name: test_authorized_for_unproctored
     type: TABLE
     table: LIVD-SARS-CoV-2
-    tableColumn: is_unproctored
+    tableColumn: Telehealth Proctor Supervised
     reference: Is the test authorized for unproctored administration by the FDA
     referenceUrl: https://www.fda.gov/news-events/fda-newsroom/press-announcements
-    mapper: livdLookup()
-    default: N
-    documentation: Is the test authorized for unproctored administration by the FDA (Y, N, UNK)
+    mapper: LivdNegateYesNo()
+    default: Y
```

Comment:
Defaulted to `Y` since in the LIVD table, a blank entry for `Telehealth Proctor Supervised` suggest it is unproctored.